fuzz/fuzz_targets/ruff_fix_validity.rs

```diff
@@ -0,0 +1,25 @@
+//! Fuzzer harness which actively tries to find testcases that cause Ruff to introduce errors into
+//! the resulting file.
+
+#![no_main]
+
+use libfuzzer_sys::{fuzz_target, Corpus};
+use ruff::settings::Settings;
+
+static mut SETTINGS: Option<Settings> = None;
```

Comment:
You can use `OnceCell` for a lazy initialized static object without relying on unsafe code https://doc.rust-lang.org/std/cell/struct.OnceCell.html

Reply:
Ah, okay. Wasn't sure of the performance implications here, but I'll give it a try.